### ISSUE TITLE
Coherent armour system: upgrades.

### ISSUE THREAD

**Issue by Dylan4ever** [ganymedes01/TheErebus#287]
UPGRADES:

- IMMOVABLE UPGRADE
Lets you take 12.5% less knockback from attacks per level.
= Can be applied to all armour pieces. 

- POISON POINT UPGRADE
Enemies that hit you with melee attacks have a 12.5% chance to get poisoned (Poison II) for 5 seconds. 
= Can be applied to all armour pieces.

- ELASTIC UPGRADE
You can jump 4 blocks high and fall damage is reduced.
= Can only be applied to your boots.

- GLIDER UPGRADE
Gives you the ability to glide.
= Can only be applied to your chestplate.
= You can not apply more than one to your chestplate.

- FLIGHT UPGRADE
Similar to the current Powered Glider Chestplate, it gives you flight.
= Can only be applied to your chestplate.
= You can not apply more than one to your chestplate.

- VELOCITY UPGRADE
You will get a speed boost similar to the current level 3 and level 6 speed leggings (depending on the amount of these that you put on your leggings). You will also walk up 1-block high inclines without jumping.
= Can only be applied to your leggings.

- SUPER VELOCITY UPGRADE
You will get a speed boost similar to the current level 6 speed leggings. You will also walk up 1-block high inclines without jumping.
= Can only be applied to your leggings.
= You can not apply more than one to your leggings. 
= If you have a Super Velocity upgrade and a normal Velocity upgrade, you will get a speed boost equal to the current level 9 sprint leggings.

- COMPOUND GOGGLES UPGRADE
It gives you a Night Vision trait, it is pretty much the current Reinforced Compound Goggles.
= Can only be applied to your helmet.
= You can not apply more than one to your helmet.

- GLOWING GEM UPGRADE
It gives the player the same amount of light emittingness as a Glowworm.
= Can only be applied to your helmet.
= You can not apply more than one to your helmet.

- WATER STRIDING UPGRADE
Basically gives you the current water striding effects.
= Can only be applied to your boots.

===============================================================================

READ THIS.

Many of the upgrades are unique to one piece of armour. The effect of the ones that aren't are calculated based on the amount of upgrades you have in all of your armour pieces.

**Comment by vadis365** [comment]
Closed due to it's not happening yet.
